CONTRIBUTING.rst

```diff
@@ -1,177 +1,228 @@
 
-**How does DevPortal work?**
+* **HowTo** walks a developer through completing a specific task. It has a clear introduction of the task and what is is for, and numbered steps to follow (without choices, if there's more than one way to do something, pick one). If additional explanation could be useful, link to a related concept article.
+    - Example: https://developer.aiven.io/docs/products/postgresql/howto/upgrade.html
+    - Template: `Howto article template`_
 
-It is Python-based, with content in ReStructuredText (rst) and rendered by Sphinx.
+* **Concept** is an explainer article that helps a developer to learn or understand something. Developers researching features, or who need extra background information on something that has been mentioned in a task, rely on these articles to fill gaps in their knowledge. It's common for HowTo articles to link to Concept articles.
+    - Example: https://developer.aiven.io/docs/products/postgresql/concepts/pg-connection-pooling.html
+    - Template: `Concept article template`_
 
+* **Reference** starts with an overview of what can be found in this section, and then usually a list of some kind. We use this type of article for lists of extensions, configuration parameters, and that sort of bookmark-it-for-reference type of content.
+    - Example: https://developer.aiven.io/docs/products/kafka/reference/advanced-params.html
 
-**What is the content model?**
+Pull request process
+--------------------
 
-DevPortal contains the following content types:
-- *Concept* - Conceptual content helps people understand a feature or topic by providing a clear, high-level overview, explanation of how the feature or topic can help them on their journey, and context like use cases or examples. 
-- *Task* - Task content helps people complete a task from start to finish while they're using Aiven’s products.
-- *Reference* - Referential content provides detailed information that people need while they're actively using Aiven's products.
-- *Tutorial* - Tutorials help people learn about products, and solve real world problems by guiding them through the entire workflow to complete a task.
+The instructions for setting up local development are in the ``README``.
 
-.. tip::
-    Use the templates included below for further guidance. 
+0. We heartily recommend you open an issue so that we can either help with outlines, or know that three people are working on the same thing.
 
-To learn more about documentation content types, see the `Diátaxis Framework <https://diataxis.fr/>`_.
+1. Make your changes on a branch. We recommend a maximum of three articles, and don't review large changesets without prior discussion. File names are ``kebab-case``, with hyphens, such as ``connect-with-pgadmin.rst``. Files should also be added to the table of contents file ``_toc.yml`` so they appear in the right location in the navigation.
```

Comment:
I'd say an article per a pull request.... Easier to review and revert if needed. Three different articles under same change are troublesome

Reply:
We had one initially but it discourages adding a supporting concept to go with your how-to so that's how I ended up at three. I don't mind four but I'm not interested in having ten!